src/makeCalculator.js

```diff
@@ -37,7 +37,33 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  const obj = {
+    result: 0,
+    add(number) {
+      obj.result += number;
+    },
+    subtract(number) {
```

Comment:
Better to add spaces to improve the readability of the code
```suggestion
    add(number) {
      obj.result += number;
    },
    
    subtract(number) {
```